Mixpanel/MixpanelAPI.cs

```diff
@@ -257,6 +259,7 @@ public static void Track(string eventName, string key, Value value)
             Value properties = new Value();
             properties[key] = value;
             Controller.DoTrack(eventName, properties);
+            MixpanelStorage.HasTracked = true;
```

Comment:
looks like this is set to true inside of `DoTrack()` on the line above, right?

Reply:
good catch